Sources/WebSocketKit/WebSocket.swift

```diff
@@ -164,21 +220,38 @@ public final class WebSocket: Sendable {
         fin: Bool = true,
         promise: EventLoopPromise<Void>? = nil
     ) {
-        let frame = WebSocketFrame(
-            fin: fin,
-            opcode: opcode,
-            maskKey: self.makeMaskKey(),
-            data: data
-        )
-        self.channel.writeAndFlush(frame, promise: promise)
+        if let p = pmce,
+               p.enabled {
```

Comment:
Why would we have a PMCE without it being enabled?

Reply:
I think maybe it's not needed now. but really wanted some more testing which seems to be unobtanium.

Reply:
working on removing it and seeing what happens.